### DISCUSSION TITLE
scan-build report location

### DISCUSSION THREAD

**Discussion by gemesa** [aircrack-ng/aircrack-ng#2509] [Q&A] [answered]
Where are the scan-build reports located? I do not see them attached as an artifact for each run. Some guides mention they can be found under the Security tab if I understand correctly. If so, I guess only the owner and collaborators can see it.

**Comment by Mister-X-** [comment, marked as answer]
They are now uploaded as artifacts (and I made it run for PR as well; it may need adjustments). Let me know if you can access them.

For anyone else reading, there seems to be a SARIF viewer plugin for VSCode.
